app/models/pawn.rb

```diff
@@ -1,4 +1,38 @@
 class Pawn < Piece
-	def move
+
+	#en passant status of self
+	def en_passant_status(end_x, end_y)
```

Comment:
this code looks off to me, first why store en_passant in the database if it can be calculated at runtime.  unless you are keeping track to make sure you can only do the en_passant move once in that case fine, gotcha store the value.  But consider naming you columns to represent the question you are asking..  instead of en_passant perhaps in the future use.  has_already_done_en_passant_move  as the column name sure it's longer but far more descriptive.

secondly, this method take two parameter end_y and end_y but only makes use of one of them.  This is a code smell that would typically require some explaining.